index.html

```diff
@@ -74,7 +74,7 @@ <h3 id="terminal-header">Ceph</h3>
         </div>
     </section>
     <h1 class="text-center">Who loves Rook?</h1>
-    <p class="text-center">With hundreds of contributors and millions of downloads of the Rook software, this true community-driven effort is putting dynamic orchestration, high performance, and solid reliability in the hands of a global user base.<br /><br />Rook is deployed in production across multiple industries, enabling them to store, deliver & protect the data that powers their businesses.</p>
+    <p class="text-center">With a hundred contributors and tens of millions of downloads of the Rook software, this true community-driven effort is putting dynamic orchestration, high performance, and solid reliability in the hands of a global user base.<br /><br />Rook is deployed in production across multiple industries, enabling them to store, deliver & protect the data that powers their businesses.</p>
```

Comment:
there's 162 contributors currently, changing this to say "a hundred" exactly sounds a bit awkward or underselling the number.

According to https://www.dictionary.com/browse/hundreds, an accepted definition is:
> hundreds, a number between 100 and 999, as in referring to an amount of money: Property loss was only in the hundreds of dollars.

I'm fine with the wording as is :)

Reply:
Where does the 162 number come from. Github reports only a hundred to me here https://github.com/rook/rook/graphs/contributors

Reply:
I guess that's a limitation of GitHub. :\

Reply:
yeah i think that list just shows the first 100 and not any others.  this is where i see 162, on the main repo page:
![Screen Shot 2019-06-13 at 9 38 20 AM](https://user-images.githubusercontent.com/4313439/59450889-1b6e6380-8dbf-11e9-907c-5d1a14d9e9b7.png)